ballerina/tests/resources/expected_results/invalid_block_strings.json

```diff
@@ -0,0 +1,13 @@
+{
+    "errors": [
+        {
+            "message": "Syntax Error: Cannot parse the unexpected character \"\"\".",
```

Comment:
This error seems to be not right.  In the given document, the block string ends at `{ line: 4, column: 1 }`.
Then there's another identifier, which means we have another argument. Then there's another identifier without a colon character, which is incorrect.

So the error should be 
```
{
    message: "Syntax Error: Expected \":\", found Name.",
    locations: [
        {
            line: 6,
            column: 14
        }
    ]
}
```